tests/Fixer/Operator/ConcatSpaceFixerTest.php

```diff
@@ -16,20 +16,40 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum
  *
  * @internal
  */
-final class ConcatWithoutSpacesFixerTest extends AbstractFixerTestCase
+final class ConcatSpaceFixerTest extends AbstractFixerTestCase
 {
     /**
-     * @dataProvider provideCases
+     * @expectedException \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException
+     * @expectedExceptionMessageRegExp #^\[concat_space\] Missing "spacing" configuration.$#
      */
-    public function testFix($expected, $input = null)
+    public function testInvalidConfigMissingKey()
     {
+        $this->getFixer()->configure(array('a' => 1));
+    }
+
+    /**
+     * @expectedException \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException
+     * @expectedExceptionMessageRegExp #^\[concat_space\] "spacing" configuration must be "one" or "none".$#
+     */
+    public function testInvalidConfigValue()
+    {
+        $this->getFixer()->configure(array('spacing' => 'tabs'));
```

Comment:
This will be a repeated pattern in the test of fixers that are configurable. Maybe we can add a protected method to the base test class to get a fresh clone of the fixer being tested?